src/bin/flareact.js

```diff
@@ -46,7 +49,7 @@ if (argv._.includes("dev")) {
       },
       {
         command:
-          "webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development",
+          "./node_modules/flareact/node_modules/.bin/webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development",
```

Comment:
This definitely solves some global webpack-dev-server issues that my team was facing. 👍